dsconfig/tangodb.py

```diff
@@ -334,6 +334,7 @@ def maybe_upper(s, upper=False):
 
 def get_servers_with_filters(dbproxy, server="*", clss="*", device="*",
                              properties=True, attribute_properties=True,
+                             class_properties=False,
```

Comment:
This argument `class_properties` is not used in this method. 
It's probably better to pop it out of the `options` dict before calling `get_servers_with_filters`